pkg/features/kube_features.go

```diff
@@ -925,7 +926,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	DynamicResourceAllocation: {Default: false, PreRelease: featuregate.Alpha},
 
-	EventedPLEG: {Default: false, PreRelease: featuregate.Alpha},
+	EventedPLEG: {Default: false, PreRelease: featuregate.Beta}, // off by default, requires CRI Runtime support
```

Comment:
Could you handle this automatically by container runtime? I mean that kubelet can check if it is supported and uses evented-pleg if it is supported and falls back to using old behavior if it is not supported.

Reply:
We already [handle this in the code](https://github.com/kubernetes/kubernetes/blob/d446bebca8008ff8ba8906c76647c0ae40dbb6f0/pkg/kubelet/pleg/evented.go#L181). However, we still want the user to explicitly set the feature gate here. This way the feature gate for this particular feature is respected and is only used when set by the user. 

The code I pointed out takes care of the situation where the user has enabled this feature gate but for some reason the runtime is unable to send any event (either the runtime doesn't support evented pleg or there is some error in the runtime).

Reply:
With the default set to false, is the Beta promotion basically telling users that this feature is more mature?